lib/ui/pointer.dart

```diff
@@ -61,7 +89,25 @@ enum PointerDeviceKind {
   /// A pointer device with a stylus that has been inverted.
   invertedStylus,
 
-  /// A touch-based pointer device with an indirect surface.
+  /// Gestures from a touch-based pointer device with an indirect surface.
+  ///
+  /// On supporting platforms, when the user is operating on a physical
+  /// trackpad, events with kind [trackpad] are dispatched when the user
+  /// performs gestures on the trackpad, such as panning, zooming, scrolling,
+  /// or rotating. The [trackpad] kind is only found with [PointerChange] `add`,
+  /// `remove`, and pan-zoom related values.
+  ///
+  /// Some platforms do not support or fully support trackpad gestures, and
+  /// might convert trackpad gestures into fake pointer events that simulate
+  /// dragging. This include Web, and iOS when `Info.plist` is not present or
+  /// returns NO for `UIApplicationSupportsIndirectInputEvents`.
+  ///
+  /// Moving the pointing cursor or clicking never triggers events with
```

Comment:
More gratuitous rewording

Moving the pointing cursor or clicking typically triggers [touch] or [mouse] events. Doing so never triggers [trackpad] events.